tests/conftest.py

```diff
@@ -53,13 +53,26 @@ def w3():
   }
 }
 
+
+@pytest.fixture
+def valid_lockfile():
+    with open("ethpm/assets/validLockfile.json") as file_obj:
+        return json.load(file_obj)
+
+
+@pytest.fixture()
+def invalid_lockfile():
+    with open("ethpm/assets/invalidLockfile.json") as file_obj:
+        return json.load(file_obj)
+
+
 @pytest.fixture
 def lockfile_with_no_deployments(tmpdir):
     lockfile = copy.deepcopy(LOCKFILE)
     lockfile.pop("deployments")
     f = tmpdir.join("lockfile.json")
     f.write(json.dumps(lockfile))
-    return str(f)
+    return json.load(f)
```

Comment:
Note what this function is currently doing.  Taking the json, writing it to a file, then returning the json parsed from the file.  You can shorten this whole function (and the subsequent fixtures) to just `return` the lockfile data and remove all of the *file* nonsense.

Reply:
Very good point. I did this from the perspective of fixing the tests and hadn't zoomed out enough to see the obvious.